pybamm/discretisations/discretisation.py

```diff
@@ -497,15 +501,66 @@ def create_mass_matrix(self, model):
 
         return pybamm.Matrix(mass_matrix)
 
+    def create_jacobian(self, model):
+        """Creates Jacobian of the discretised model.
+        Note that the model is assumed to be of the form M*y_dot = f(t,y), where
+        M is the (possibly singular) mass matrix. The Jacobian is df/dy.
+
```

Comment:
Add a comment here referring to this PR to explain why this function isn't used. I could see us forgetting and wondering why we don't use this in future

Reply:
also, if we make some kind of master simulation class we could use this method from the disretisation in the solver set up since we would be able to access the disc of the model when in the solver set up